xml/System/Memory`1.xml

```diff
@@ -137,10 +190,20 @@
         <Parameter Name="other" Type="System.Memory&lt;T&gt;" />
       </Parameters>
       <Docs>
-        <param name="other">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="other">The object to compare with the current instance.</param>
+        <summary>Determines whether the specified <see cref="T:System.Memory`1" /> object is equal to the current object.</summary>
+        <returns><see langword="true" /> if the current instance and <paramref name="other" /> are equal; otherwise, <see langword="false" />. </returns>
+        <remarks>
```

Comment:
We should probably add a remark to clarify that this is purely reference equality and we are not comparing the elements of `Memory<T>` for equality.